DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: EpiEstim
-Version: 2.2-1
+Version: 2.2-1.9000
```

Comment:
I would just bump the version up a minor number, personally
```suggestion
Version: 2.2-2
```

Reply:
I think that's a good point. Increasing the version per patch makes sense for tracking how people got the code.